qutebrowser/browser/pdfjs.py

```diff
@@ -44,6 +44,63 @@ def __init__(self, path):
         super().__init__(message)
 
 
+def _run_js(browsertab, code):
```

Comment:
I don't think this is needed. There is already `JsWorld` in `utils.usertypes` which is backend-agnostic, and you can pass a world ID to `webkittab` as well. It does log a warning though, but I guess this warning could as well just be removed...